src/main/java/graphql/language/NodeUtil.java

```diff
@@ -77,7 +77,7 @@ public static GetOperationResult getOperation(Document document, String operatio
         }
         OperationDefinition operation;
 
-        if (operationName == null || operationName == "") {
+        if (operationName == null || "".equals(operationName)) {
```

Comment:
Another better way to do this is 

    if (operationName == null || operationName.isEmpty()

Reply:
I am going to accept this PR (because the old code is wrong) but if you could make another one then that would be great

Reply:
Oh.  Thank you for teaching me !   
I will change the code as you advised me  !